lib/zendesk_apps_support/manifest.rb

```diff
@@ -101,6 +103,15 @@ def set_locations_and_hosts
         end
     end
 
+    def set_framework_options
+      @framework_options ||= {
+        noTemplate: no_template_locations,
+        singleInstall: single_install?,
```

Comment:
hmm, well maybe this naming scheme doesn't work that well then...
https://github.com/zendesk/zendesk_app_framework/blob/01f77fca6d3b00684d47706205d23bd7c8beabb9/lib/app/base.js#L123